01_fibonacci/fibi.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+//prints the fibonacci series
+func fibonacci(n int) {
+	a, b := 0, 1
+
+	for i := 0; i <= n; i++ {
+		if a >= 1 {
+			fmt.Fprintln(out, a)
+		}
+		a, b = b, (a + b)
```

Comment:
Parentheses not required. Go's [precedence rules](https://golang.org/ref/spec#Operator_precedence) are remarkably simply, btw. It's essentially: multiplicative, additive, comparison, `&&`, `||`.